File: C2/PDOO/Practicas/DeepSpaceRuby/lib/PowerEfficientSpaceStationToUI.rb
#encoding:utf-8

require_relative 'SpaceStationToUI'

module Deepspace;
  
  # 23.5.17 - Translation from Java
  # @author Profesor
  
class PowerEfficientSpaceStationToUI < SpaceStationToUI 
  def initialize (efficientStation) 
    super(efficientStation)
  end
  
  #Override
  def name
    out = super
    out += " (ESTACIÓN EFICIENTE)"
    return out
  end

end # class

end # module

File: C2/PDOO/Practicas/DeepSpaceJava/src/deepspace/PowerEfficientSpaceStationToUI.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package deepspace;

/**
 *
 * @author Profesor
 */
public class PowerEfficientSpaceStationToUI extends SpaceStationToUI {
  PowerEfficientSpaceStationToUI (PowerEfficientSpaceStation efficientStation) {
    super(efficientStation);
  }
  
  @Override
  public String getName () {
    return super.getName() + " (ESTACIÓN EFICIENTE)";
  }

}
